sign/sign.go

```diff
@@ -124,7 +124,41 @@ func (s *Signer) SignImage(reference string) (*SignedObject, error) {
 func (s *Signer) SignFile(path string) (*SignedObject, error) {
 	s.log.Infof("Signing file path: %s", path)
 
-	// TODO: unimplemented
+	resetFn, err := s.enableExperimental()
+	if err != nil {
+		return nil, err
+	}
+	defer resetFn()
+
+	ko := sign.KeyOpts{
+		KeyRef:     s.options.PrivateKeyPath,
+		PassFunc:   s.options.PassFunc,
+		FulcioURL:  cliOpts.DefaultFulcioURL,
+		RekorURL:   cliOpts.DefaultRekorURL,
+		OIDCIssuer: cliOpts.DefaultOIDCIssuerURL,
+
+		InsecureSkipFulcioVerify: false,
```

Comment:
Hm, just a nit but we can remove this line since it defaults to false.

Reply:
I was keeping it for explicitness. I can remove if this sounds weird.